### ISSUE TITLE
[frontend] Advanced code problem deprecated

### ISSUE THREAD

**Issue by ClemCordier** [UCL-INGI/INGInious#892]
[The following section](https://docs.inginious.org/en/latest/teacher_doc/task_file.html#advanced-code-problem) (Teacher's documentation &rarr; Task description files) shows a brief example of an advanced task composed of "boxes" that could optionally provide restriction on the # of characters/lines for a given box_id.

It has apparently been discontinued since then.

I guess an alternative to this functionality would be to manually check for the chars/lines submitted by students through the run file directly.

**Comment by anthonygego** [comment]
Indeed, the feature has been converted to a plugin for retrocompatibility needs : https://github.com/UCL-INGI/INGInious-problems-custom

It had quickly shown its limits in terms of task writing complexity and UI stability (as it relies on pure HTML code that can brake more easily with framework updates). Designing specific problem types that allows the course administrator to define the tasks more easily is the recommended way now.

I keep this issue open as the docs need an update.

**Comment by nrybowski** [comment]
There is also some legacy code that should be removed. See https://github.com/UCL-INGI/INGInious/search?q=maxChars.
